pygpu/gpuarray.pyx

```diff
@@ -1695,6 +1704,20 @@ cdef class GpuArray:
 
         array_setarray(tmp, gv)
 
+    def take1(self, GpuArray idx):
```

Comment:
Add tests for that method. It can be basic test just to make sure it work and we don't remove/break it by mistake.